src/Engine/AudioHandler/AudioHandler.cpp

```diff
@@ -0,0 +1,60 @@
+#include <string>
+#include <stdio.h>
+
+#include <glog/logging.h>
+#include <SDL.h>
+
+#include "AudioHandler.h"
+
+
+AudioHandler::AudioHandler() {
+
+}
+AudioHandler::~AudioHandler() {
+}
+void AudioHandler::init() {
+    //TODO only load really important things that need early access Like... Splash screen music or w/e
+}
+Mix_Music* AudioHandler:: load_music(std::string path) {
+    LOG(FATAL) << "Function should not be used yet";
+    //Mix_Music* ret = Mix_LoadMUS(path.c_str());
+
+    //if ( ret == NULL ) {
+    //LOG(FATAL) << "Fatal error, could not load music: " << path.c_str() <<
+    //"Error: " << Mix_GetError() << " Exiting...";
+    //}
+
+    //return ret;
+}
+Mix_Chunk* AudioHandler:: load_chunk(std::string path) {
+    Mix_Chunk* ret = NULL;
+    auto iterator = sound_effects_.find(path);
+
+    if ( iterator != sound_effects_.end()) {
+        ret = iterator->second;
+    } else {
+        ret = Mix_LoadWAV(path.c_str());
+
+        if ( ret == NULL ) {
+            LOG(FATAL) << "Fatal error, could not load chunk: " << path.c_str() <<
+                       "Error: " << Mix_GetError() << " Exiting...";
+        }
+
+        sound_effects_[path.c_str()] = ret;
+    }
+
+    return ret;
+
+}
+void AudioHandler::play_music(Mix_Music* music) const {
+    LOG(FATAL) << "Function should not be used yet";
+    //TODO just keeping code here
+    //if (Mix_PlayMusic(music, -1) == -1) {
+    //printf("Mix_PlayMusic: %s\n", Mix_GetError());
+    //}
+}
+void AudioHandler::play_chunk(Mix_Chunk* chunk) const {
+    if (Mix_PlayChannel(-1, chunk, 0) == -1) {
```

Comment:
What's the 0 and -1? I think would make this more readable to encapsulte those in variables so I know what they mean without looking up SDL_Mixer docs.

Reply:
oh ok. Yea i forgot what the 0 does, but the first argument is channel.

You can select a channel from 0-however many chanels we set (which is 4 i think). But if we don't want to choose a certain channel, we can just do -1 to let it automatically decide.

But yea i'll make it more readable

Reply:
I think since the config stuff is loaded, we should add be able to make that configurable now.